### ISSUE TITLE
Table does not get blocked

### ISSUE THREAD

**Issue by stroodl** [vanadiuz/table-reservation#19]
Hello,
I've been testing this great plugin and only added some translations in the table-reservation.php file. All worked great but it suddenly stopped "blocking" the tables - when I make a reservation, the table does not turn red and can be selected again. When confirming the next reservation I get an error that "Someone has already reserved the table...".
The thing is that I tried removing the plugin and installing fresh files, I even uploaded a website backup dating before I installed the plugin for the first time. Nothing has helped so I suspect some db entry causing this issue.
You can try it under https://www.penzionjezero.cz/rezervace-stolu/

Any help would be much appreciated.

Thank you,
Ondra

**Comment by vanadiuz** [comment]
Hi Ondrai,
thanks for kind words. 
I just checked it and couldn't reproduce the bug. 
Could you please make a small movie which will demonstrate this bug?

**Comment by stroodl** [comment]
Hi vanadiuz,
I've found the "bug" - the plugin doesn't like the dddd time format. Even though the date was displayed correctly in the datepicker and the reservation form and the reservation was created, it was not recognized once the code checks for booked tables (can't remember where exactly I found it). 

Nice plugin btw, I'm running it here: https://www.penzionjezero.cz/rezervacestolu/

Any chance we will be able to do more layouts for the same tables? I would like to have a very simple UI for smartphones (just the rectangle tables) plus to keep the current layout for desktop.

Thank you,
Ondra
